.github/workflows/push.yml

```
@@ -0,0 +1,62 @@
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: |
+            ~/.npm
+            node_modules
+          key: ${{ runner.os }}-code-coverage-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-code-coverage-${{ env.cache-name }}-
+            ${{ runner.os }}-code-coverage-
+            ${{ runner.os }}-
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)